Devices/GPS/Type/GpsMeasurement_t.hpp

```diff
@@ -9,8 +9,11 @@
 #include <string>
 #include <vector>
 #include <sstream>
+
+#include <SiBase.hpp>
+#include <SiExtended.hpp>
```

Comment:
Mir ist nicht direkt klar wann man welchen header braucht.
Macht es da sinn einen einzigen Header zu machen der alles enthält, für ease of use?
Ich sehe das problem, dass man dann unnötiges included, aber das passiert auch wenn man das manuell alles hinzufügt und nicht weiß wann man etwas wieder entfernen kann...

Reply:
Da kann man natürlich einen Header nehmen der nur alle anderen inkludiert. Ist einfacher zu nutzen, hat halt den Nachteil das man immmer alles inkludiert und die Build-Zeiten halt entsprechend sind.

Reply:
Es gibt jetzt einen `Si.hpp` Header der alle relevanten Header inkludiert. Hier würde ich das aber aufgrund der Buildzeit so lassen